src/modules/tests/tests.effects.ts

```diff
@@ -133,7 +133,7 @@ export class TestsEffects {
     switchMap(([action, journal, rekeySearch]: [testActions.StartTest, JournalModel, RekeySearchModel]) => {
       const startTestAction = action as testActions.StartTest;
       const isRekeySearch = this.navigationStateProvider.isRekeySearch();
-      const employeeId = this.authenticationProvider.getEmployeeId();
+      const employeeId = this.authenticationProvider.getEmployeeId() || journal.examiner.staffNumber;
```

Comment:
With the absence of unit tests, assuming this has all been tested and verified manually?

Reply:
Originally I created a helper method that wrapped this change - and had unit tests to prove that the correct value was returned if staffNumber was populated and if it was null. But realised that all I was really testing was that fundamental javascript behaviour worked as expected so there was no value to the test. 

Tested and verified manually.